service/gcs/core/gcs/storage.go

```diff
@@ -32,51 +33,87 @@ const (
 	mappedDiskMountTimeout = time.Second * 2
 )
 
-// getLayerDevices turns a list of SCSI codes (0-0, 0-1, etc.) into a list of
-// the devices (sda, sdb, etc.) that they correspond to.
-func (c *gcsCore) getLayerDevices(scratch string, layers []prot.Layer) (scratchDevice string, layerDevices []string, err error) {
-	layerDevices = make([]string, len(layers))
+type mountSpec struct {
+	Source     string
+	FileSystem string
+	Flags      uintptr
+	Options    []string
+}
+
+// Mount mounts the file system to the specified target.
+func (ms *mountSpec) Mount(target string) error {
+	options := strings.Join(ms.Options, ",")
+	err := syscall.Mount(ms.Source, target, ms.FileSystem, ms.Flags, options)
+	if err != nil {
+		return errors.Wrapf(err, "mount %s %s %s %x %s", ms.Source, target, ms.FileSystem, ms.Flags, options)
+	}
+	return nil
+}
+
+// getLayerMounts computes the mount specs for the scratch and layers.
+func (c *gcsCore) getLayerMounts(scratch string, layers []prot.Layer) (scratchMount *mountSpec, layerMounts []*mountSpec, err error) {
+	layerMounts = make([]*mountSpec, len(layers))
 	for i, layer := range layers {
-		deviceID := layer.Path
-		deviceName, err := c.deviceIDToName(deviceID)
+		deviceName, pmem, err := deviceIDToName(layer.Path)
 		if err != nil {
-			return "", nil, err
+			return nil, nil, err
+		}
+		options := []string{"noload"}
```

Comment:
Should you `const` these strings so we could know the options? Or is this just something that everyone but me knows :)